draft-ietf-taps-impl.md

```diff
@@ -739,7 +739,7 @@ Connection Properties. A Connection can also generate events in the form of Soft
 
 The set of Connection Properties that are supported for setting and getting on a Connection are described in {{I-D.ietf-taps-interface}}. For
 any properties that are generic, and thus could apply to all protocols being used by a Connection, the Transport System should store the properties
-in a generic storage, and notify all protocol instances in the Protocol Stack whenever the properties have been modified by the application.
+in an object common to all protocols, and notify all protocol instances in the Protocol Stack whenever the properties have been modified by the application.
```

Comment:
"an object" sounds odd to me here. I think it needs to store it in "the Connection Object" if we want to express it at this abstraction level.

Reply:
```suggestion
in storage common to all protocols, and notify all protocol instances in the Protocol Stack whenever the properties have been modified by the application.
```